internal/store/store.go

```diff
@@ -400,6 +400,44 @@ func (s *Store) DeleteBlueprintFromWorkspace(name string) {
 	})
 }
 
+// TagBlueprint will tag the most recent commit
+// It will return an error if the blueprint doesn't exist
+func (s *Store) TagBlueprint(name string) error {
+	return s.change(func() error {
+		_, ok := s.Blueprints[name]
+		if !ok {
+			return errors.New("Unknown blueprint")
+		}
+
+		// Get the latest revision for this blueprint, and the most recent commit
+		var revision int
+		var timestamp string
+		var commit blueprint.Change
+		for _, c := range s.BlueprintsChanges[name] {
+			if c.Revision != nil && *c.Revision > revision {
+				revision = *c.Revision
+			}
+			if c.Timestamp > timestamp {
```

Comment:
As mentioned in a later commit, timestamps may not be unique. Would it make sense to order by version as a secondary key in case the timestamps match?

Reply:
That would probably work well enough. It would only happen if the timestamps were equal, which may results in unexpected ordering already. It would only fail if the store was expecting a lower version in the same second, like a push followed by an undo.

Reply:
Yeah, not ideal, but until we have a stable release maybe acceptable?

Reply:
Thinking more about this, I would leave the sorting by version out. It is possibly improving an edge case, but not necessarily, and it can even make it worse. So adding it in here would be misleading.